types/react-ga-tests.ts

```diff
@@ -134,7 +134,7 @@ describe("Testing react-ga v2.1.2", () => {
 
         execute("name", "action", payload);
         execute("name", "action", "type", payload);
-        require("name", {});
+        require("name", {}, 'trackerName');
```

Comment:
It would be better to leave this test as is and create a sibling test for the trackerName.

Reply:
Done.